tests/functional/i/inherit_non_class.py

```diff
@@ -1,8 +1,8 @@
 """Test that inheriting from something which is not
 a class emits a warning. """
 
-# pylint: disable=import-error, invalid-name, using-constant-test, useless-object-inheritance
-# pylint: disable=missing-docstring, too-few-public-methods
+# pylint: disable=import-error, invalid-name, using-constant-test
+# pylint: disable=missing-docstring, too-few-public-methods, useless-object-inheritance

```

Comment:
```suggestion
# pylint: disable=missing-docstring, too-few-public-methods

```

Reply:
I think this one is necessary because of what [this test] is trying (https://github.com/PyCQA/pylint/blob/main/tests/functional/i/inherit_non_class.py#L27)